src/contract_wrappers/token_wrapper.ts

```diff
@@ -21,14 +21,15 @@ import {
     ContractEventObj,
 } from '../types';
 
-const ALLOWANCE_TO_ZERO_GAS_AMOUNT = 45730;
+const ALLOWANCE_TO_ZERO_GAS_AMOUNT = 47155;
 
 /**
  * This class includes all the functionality related to interacting with ERC20 token contracts.
  * All ERC20 method calls are supported, along with some convenience methods for getting/setting allowances
  * to the 0x Proxy smart contract.
  */
 export class TokenWrapper extends ContractWrapper {
+    public UNLIMITED_ALLOWANCE_IN_BASE_UNITS = new BigNumber(2).pow(256).minus(1);
```

Comment:
What we did with the other exported constant is still define it in `constants` and simple export it here. Let's stick with that convention.